src/Compilers/CSharp/Test/Syntax/Parsing/ParserRegressionTests.cs

```diff
@@ -13,6 +13,24 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Parsing
 {
     public class ParserRegressionTests : CSharpTestBase
     {
+        [Fact]
+        public void PartialLocationInModifierList()
+        {
+            var comp = CreateStandardCompilation(@"
+class Program
+{
+    partial abstract class A {}
+    partial abstract class A {}
+
+    partial partial class B {}
+    partial partial class B {}
+
+    partial abstract struct S {}
+    partial abstract struct S {}
+}");
+            comp.VerifyDiagnostics();
```

Comment:
I didn't understand why there are no diagnostics. Shouldn't the CheckModifiers call report that partial is misplaced?

Reply:
Ah, this should fail now. I may have forgotten to update the baseline.